index.html

```diff
@@ -27,7 +27,6 @@
       <div class="logo">
         <img class="img-logo" src="images/logo.svg" alt="logo">
         <img class="bank-logo" id="hamburger" src="images/icon-hamburger.svg" alt="icon-hamburger">
-        <!-- <img src="images/icon-close.svg" alt="icon-close" id="close-btn"> -->
       </div>
       <nav class="navbar ">
```

Comment:
:red_circle: Please make sure the height of your top bar is similar to the one in the design. As you can see in these two screenshots, your top bar has some additional height. Kindly reduce the height of your top bar.
Your Design
![topbar](https://github.com/aradradev/easyBankTC/assets/26736582/5f60e1bd-3aec-44fe-810b-1d624507048f)

Template Design

![topbar-ok](https://github.com/aradradev/easyBankTC/assets/26736582/9c69a2e1-2153-47e5-8133-70456d5c9598)